rules/signing.bzl

```diff
@@ -948,3 +948,46 @@ keyset = rule(
         ),
     },
 )
+
+def _signature_test_impl(ctx):
+    _script = ctx.actions.declare_file("{}.bash".format(ctx.label.name))
+
+    files = [f.short_path for f in ctx.files.srcs]
+    joined = " ".join(files)
+    ctx.actions.expand_template(
+      template = ctx.file._script,
+      output = _script,
+      substitutions = {
+        "@FILES@": joined,
+        "@OPENTITANTOOL@": ctx.executable._opentitantool.short_path,
+      },
+      is_executable=True,
+  )
+
+    return [
+        DefaultInfo(
+            runfiles = ctx.runfiles(files=ctx.files.srcs + [ctx.executable._opentitantool]),
+            executable= _script,
+        )
+    ]
+
+signature_test = rule(
+  implementation = _signature_test_impl,
+  attrs = {
+      "srcs": attr.label_list(allow_files=True, doc="help string"),
+      "_script": attr.label(
+            default = Label("//rules/scripts:sival_signature_test.bash"),
```

Comment:
You don't need an explicit label constructor here.  The `attr.label` function should convert the supplied `default` string to a label for you (here, and below for `_opentitantool`).